user_guide_src/source/helpers/text_helper.rst

```diff
@@ -30,7 +30,8 @@ The following functions are available:
     Generates a random string based on the type and length you specify.
     Useful for creating passwords or generating random hashes.
 
-    .. warning:: Except for type **numeric** and **crypto**, no cryptographically secure
+    .. warning:: Except for type **alpha**, **alnum**, **numeric**, **nozero**,
+        and **crypto**, no cryptographically secure
         strings are generated. Therefore, it must not be used for cryptographic
         purposes or purposes that requires return values to be unguessable.
```

Comment:
```suggestion
    .. warning:: For types: **md5**,  and **sha1**, generated strings are no 
        cryptographically secure. Therefore, these types cannot be used for cryptographic
        purposes or purposes requiring unguessable return values.
```

Reply:
Added **basic** and updated: 80ea505d28d590fee553602e583ffe10f593da2d

Reply:
Thanks!